lib/puma/plugin/yabeda_prometheus.rb

```diff
@@ -30,11 +30,19 @@ def start(launcher)
     events.on_booted do
       app = Yabeda::Prometheus::Exporter.rack_app(Yabeda::Prometheus::Exporter, path: path)
 
-      metrics = Puma::Server.new app, events
-      metrics.min_threads = 0
-      metrics.max_threads = 1
+      metrics = Puma::Server.new app, events, min_threads: 0, max_threads: 1
```

Comment:
Puma upgrading notes says:

> If you are creating your own Puma::Server objects, it's initialize signature has changed. In 5.x and below, it was `def initialize(app, events=Events.stdio, options={})`. Now it is `def initialize(app, log_writer=LogWriter.stdio, events=Events.new, options = {})`.

So, initialization should be changed somehow…

Reply:
Oh, I see, this change was actually reverted in https://github.com/puma/puma/pull/2965, they just forgot to remove entry from upgrading notes.